src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.design

```diff
@@ -1,4 +1,4 @@
-<design:component>
+<design:component label="Batch Gift Entry Configuration">
```

Comment:
The Wizard doesn't need one, since it is only invoked from the tab or a batch record home. It had flexipage:availableForRecordHome because it used to be available on Template record home. So that can come out now.
I mentioned the BGE_BatchGiftEntryTab needing a design file, but I don't think it does, since we are shipping the tab and it doesn't appear in App Builder.